src/Microsoft.ML.Transforms/Text/WordTokenizing.cs

```diff
@@ -444,7 +444,8 @@ public sealed class ColumnOptions
         {
             public readonly string Name;
             public readonly string InputColumnName;
-            public readonly char[] Separators;
+            public IReadOnlyList<char> Separators => SeparatorsArray;
```

Comment:
Same question as above - how can users specify this?

Reply:
Same answer.